### ISSUE TITLE
mscolab server command crashes

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#491]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/491) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

There is an error with respect to the “app” variable missing. Changing the code of the stub to

‌

```
#!/home/icg173/anaconda3/envs/mss-1.9.0/bin/python
  
# -*- coding: utf-8 -*-
import re
import sys

from mslib.mscolab.server import main, app

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(main(app))
```

Fixes the problem, but we need a clean solution that generates a\) the proper stub or b\) doesn’t require the app argument in the call to main\(\).

**Comment by ReimarBauer** [comment]
**Original comment by Shivashis Padhi (Bitbucket: [plant99](https://bitbucket.org/plant99), GitHub: [plant99](https://github.com/plant99)).**

----------------------------------------

Yes, I can do  
`global app`  
to fix this.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in plant99/mss/i491 (pull request #744)

Fixes issue#491 issue#483
